test/purchases_flutter_e2e.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
+import 'package:purchases_flutter/purchases_flutter.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Can setup Purchases', (WidgetTester tester) async {
+    final Future<void> future = Purchases.setup('apiKey', appUserId: 'cesar');
+    expect(future, completes);
+  });
+
+}
```

Comment:
Also copied from the upgrading docs. I ran it by doing `./gradlew app:connectedAndroidTest -Ptarget=`pwd`/../../test/purchases_flutter_e2e.dart`inside `example/android` and it works. In Android Studio, it doesn't run I don't know why.

Also saw another example here https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_core/firebase_core/test/firebase_core_e2e.dart

This test actually runs in an emulator, so my understanding is we could test a bunch of stuff this way. It only works in Android though. I tried making a purchase and I could see the purchase dialog popping up :)

I am not even adding it to circleci for now, but I am going to leave it since it's part of the upgrading docs.

Reply:
maybe we could have either a comment or something documenting how to actually run this test so we don't forget? or even a gradle task that wraps the command